example/scripts/minio_core_loader.py

```diff
@@ -0,0 +1,214 @@
+
+
```

Comment:
Unnecessary whitespace.

Reply:
Also, why is this file called `minio_core_loader`? The `yugabyte_core_loader` was loading Core files specifically, but in this case we're loading all sorts of CSVs in minio -- can we call this `minio_csv_loader`?